src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/CSYamlModelGenerator.cs

```diff
@@ -870,16 +870,22 @@ private static ExpressionSyntax GetLiteralExpression(TypedConstant constant)
 
         private static ExpressionSyntax GetLiteralExpression(object value, ITypeSymbol type)
         {
+            bool isNullable = type.GetDocumentationCommentId().StartsWith("T:System.Nullable{");
```

Comment:
Is there a better way of detecting nullable value types? Using docId feels like a hack. `type.OriginalDefinition` seems to be always `System.Nullable<T>`?

Reply:
Not sure - I went with the technique that was currenty being used everywhere else to for type tests:

https://github.com/mikernet/docfx/blob/5150c5f87ee7bbf9922344dd4edb910a1aea8bee/src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/CSYamlModelGenerator.cs#L898

https://github.com/mikernet/docfx/blob/5150c5f87ee7bbf9922344dd4edb910a1aea8bee/src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/CSYamlModelGenerator.cs#L1136

https://github.com/mikernet/docfx/blob/5150c5f87ee7bbf9922344dd4edb910a1aea8bee/src/Microsoft.DocAsCode.Metadata.ManagedReference.Roslyn/Visitors/CSYamlModelGenerator.cs#L1157

Etc...

Reply:
Then I'm good if this is an established way.